tests/util-tests.scm

```diff
@@ -1,14 +1,16 @@
-(define-module (tests util-test)
-    #:use-module (srfi srfi-64)
-    #:use-module (opencog)
-    #:use-module (opencog exec)
-    #:use-module (opencog bioscience)
-    #:use-module (annotation util)
-
+#! /usr/bin/env guile
+!#
+
+(use-modules
+	(srfi srfi-64)
+	(opencog)
+	(opencog exec)
+	(opencog bioscience)
+	(annotation util)
 )
 
 (test-begin "util")
-(primitive-load-path "tests/sample_dataset.scm")
+(primitive-load "../tests/sample_dataset.scm")
```

Comment:
Same as the above comment. Revert the path string should be reverted to its previous version without `..`